README.md

```diff
@@ -102,8 +108,9 @@ Example (to be placed into your Yii2 config file's
     ],
 
 ### If using syslog to send to Logentries, only send the JSON content
-Make sure that the template you define for Logentries in your rsyslog.conf file 
-does not add other content before the ```%msg%``` data (aside from your 
+
```

Comment:
I really dislike wasting vertical space like this, but it would seem this is more the standard.

Reply:
Yeah, I agree. But standardization almost always trumps personal preference.